src/brian.py

```diff
@@ -266,11 +273,11 @@ def main():
     parser = argparse.ArgumentParser()
     help_msg = 'Run only the router. Do not run any of the other threads or functions.'
     parser.add_argument('--router-only', action='store_true', help=help_msg)
+    parser.add_argument("--realtime-off", action="store_true", help="Flag if realtime is disabled")
```

Comment:
Where is this used within brian?

Reply:
Forgot to delete it, I was using this to determine whether to drop messages. On further thought, this could be passed to `router()` to determine how to log a dropped message (log as warning if realtime is supposed to be running, otherwise log as a debug message).